### DISCUSSION TITLE
Welcome to btc-ticker Discussions!

### DISCUSSION THREAD

**Discussion by holgern** [btc-ticker/btc-ticker#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by iosx-Danny** [comment]
Hi There,

I need a little help.. 
The ₿ Ticker works fine with the 2,7" Display. 
I want to upgrade to the 7,5" display, but don't know how I change the settings... can anybody help me with that? 

thanks!!

**Comment by MaAlte** [comment]
Hello, 
perhaps I didn't found it in the configfile. 
Is there a Possibility to set the refreshrate of the Ticker to a longer interval?

Geets MaAlte

**Comment by Muftijaja** [comment]
Hi Holger, 
I couldn't find your ebay and kleinanzeigen links (1.10.25) anymore, thats sad, but after a deeper search I found the 3D printing data for that housing. You can find it at https://grabcad.com/library/btc-ticker-enclosure-1   Greetz Hanno

**Comment by Muftijaja** [comment]
Hello again! I got my ticker running yesterday, thanks for this tiny project! 
1. moscowtime in the config.in seems to be the wrong timezone. What to do when I live in Hamburg, Germany? berlintime didn't work.
2. changing password is not possible with your lines.
Thanks for your help! Hanno
